Dockerfile

```
@@ -15,7 +15,7 @@ COPY bin/rds-ca-rsa4096-g1.pem /bin/rds-ca-rsa4096-g1.pem
 COPY bin/rds-ca-2019-root.pem /bin/rds-ca-2019-root.pem
 COPY bin/milmove /bin/milmove
 
-COPY config/tls/Certificates_PKCS7_v5.6_DoD.der.p7b /config/tls/Certificates_PKCS7_v5.6_DoD.der.p7b
+COPY config/tls/milmove-cert-bundle.p7b /config/tls/Certificates_PKCS7_v5.6_DoD.der.p7b
```

```suggestion
COPY config/tls/milmove-cert-bundle.p7b /config/tls/milmove-cert-bundle.p7b
```

I don't think we should keep the name the same here for what gets picked up in the application. I know this way of doing things requires more app changes in the env files. But ultimately it is clearer to folks working through the code to see that this is no longer exclusively a DoD certificate bundle.